src/icons/dots-vertical.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
```

Comment:
this name in EDS is `"more-vert"` which I really hate 😂  there's also already an icon in this branch named `dots-horizontal` so this is a nice parallel

Reply:
OR we rename `dots-horizontal` to `more-hor`